### PR TITLE
Mind Control suddenly wakes up, as though he was under foreign control!

### PR DESCRIPTION
**Pull request by ghost**
image

## Changelog
:cl:
fix: You no longer see "Mind Control suddenly wakes up, as though he was under foreign control!" when a hivemind host mind controls someone
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
Amazing.

File: code/modules/spells/spell_types/hivemind.dm [hunk 1]
```diff
@@ -389,7 +389,7 @@
 			backseat.ghostize(0)
 		else
 			backseat.mind.transfer_to(vessel,1)
-	vessel.visible_message("<span class='userdanger'>[src] suddenly wakes up, as though he was under foreign control!</span>")
+	vessel.visible_message("<span class='userdanger'>[vessel] suddenly wakes up, as though he was under foreign control!</span>")
```
Comment (nemvar) [inline code/modules/spells/spell_types/hivemind.dm]: I'm no english major, so I don't know if using the past tense is actually correct in the latter part of this message, but it certainly isn't correct to always use the male genus.
```
	vessel.visible_message("<span class='userdanger'>[vessel] suddenly wakes up, as though [vessel.p_they()] [vessel.p_were()] under foreign control!</span>")

**Comment by wesoda25** [comment]
Sentence is still pretty bad. Something more along the line of "[Vessel]'s body jerks sharply as they regain control of their limbs" but that sentence is kinda meta aibhdgvajbjsndkm

**Comment by Tlaltecuhtli** [comment]
use the normal danger span thing, big ass text is used ONLY for really important shit like being shot or ai blowing machinery near you, not to prevent john mc tider from being valided by sec

**Comment by nemvar** [comment]
Someone not getting valided is fairly important, don't you think? Not like mind control happens every 30 seconds.

**Pull request merged**